docs/user-guide/features.rst

```diff
@@ -33,6 +33,27 @@ Administrators can also :ref:`upload <create-errata-recipe>` their own errata to
 a repository. Please see the :doc:`recipes` documentation to learn how to
 perform these operations.
 
+Boolean dependencies
+^^^^^^^^^^^^^^^^^^^^
+
+Pulp supports RPM content with `boolean dependencies
+<http://rpm.org/user_doc/boolean_dependencies.html>`_ in the ``Requires`` field
+in these basic repository mirroring use cases:
```

Comment:
i am not sure if publication or upload goes under mirroring action. maybe replace it with repository management or content management?

Reply:
cool :)